bitnami/kube-prometheus/templates/exporters/kube-controller-manager/endpoints.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.kubeControllerManager.enabled .Values.kubeControllerManager.endpoints }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: kube-system
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+    k8s-app: kube-controller-manager
```

Comment:
I think it is not needed, see `kube-prometheus/templates/exporters/kube-proxy/endpoints.yaml`

Reply:
labels corrected

Reply:
you mean the k8s-app is not needed or the whole prefix related to helm release?
So the question is - taking into consideration that there are "common labels" what is needed is only (as is in kube-proxy):
    app: kube-controller-manager
or we need the release included:
    app: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager

This is question for all the objects created by the chart...

Reply:
We would need to have `app.kubernetes.io/component: kube-controller-manager`, don't you think?

Reply:
yes, agree - let's keep the endpoints labels as in kube-proxy.
Corrected.